src/pages/leejiyon/Login/Login.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import './login.scss';
+
+class Login extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      id: '',
+      pw: '',
+    };
+  }
+
+  onChangeInput = e => {
+    this.setState({
+      [e.target.name]: e.target.value,
+    });
+  };
```

Comment:
구조 분해 할당 이용하셔서 아래와 같이 쓰시면 좋을 것 같아요.

```suggestion
  onChangeInput = e => {
    const { name, value } = e.target;
    
    this.setState({ [name]: value });
    };
```

Reply:
👍👍